.travis.yml

```diff
@@ -3,7 +3,7 @@ language: go
 go_import_path: github.com/pingcap/tidb
 
 go:
-  - "1.11.2"
+  - "1.11.3"
```

Comment:
irrelevant change?

Reply:
oh, this afternoon our Travis CI have a error.

Reply:
so we have to update `go` to fix that?

Reply:
The conclusion is that the go 1.11.4 can't compile tidb successfully